backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/handlers/InvokeSingleClickOnTwoDiagramElementsToolEventHandler.java

```diff
@@ -66,13 +70,20 @@ public class InvokeSingleClickOnTwoDiagramElementsToolEventHandler implements ID
 
     private final ICollaborativeDiagramMessageService messageService;
 
+    private final IRepresentationDescriptionSearchService representationDescriptionSearchService;
+
+    private final IConnectorToolsProviderRegistry connectorToolsProviderRegistry;
+
     private final Counter counter;
 
-    public InvokeSingleClickOnTwoDiagramElementsToolEventHandler(IObjectService objectService, IDiagramQueryService diagramQueryService, IToolService toolService, ICollaborativeDiagramMessageService messageService,
-            MeterRegistry meterRegistry) {
+    public InvokeSingleClickOnTwoDiagramElementsToolEventHandler(IObjectService objectService, IDiagramQueryService diagramQueryService, IToolService toolService,
```

Comment:
I fail to see why this class should be modified. The mutation sent while using a tool from the connect tool or from the regular palette should be the same. Why does that need to change?